tests/data/parsers/test_parser_classification_rule.py

```diff
@@ -220,19 +224,51 @@ def test_feedback_for_criteria_with_gaps_and_overlap(
                 [2, "<0", "<0", ">10"],
                 [3, "-", "-", "-"],
             ],
-            """For conditions: (varA: <0, varB: <0). Gap for variable varC in range -inf:0.0\nFor conditions: (varA: <0). Gap for variable varB in range 0.0:inf\nOverlap for variable varA in range -inf:0.0""",
+            [
+                call(
+                    """For conditions: (varA: <0, varB: <0). Gap for variable varC in range -inf:0.0\nFor conditions: (varA: <0). Gap for variable varB in range 0.0:inf\nOverlap for variable varA in range -inf:0.0"""
+                )
+            ],
+        ),
+        (
+            [
+                ["output", "MIN_water_depth_mNAP", "MAX_flow_velocity", "MAX_chloride"],
+                [1, "<0.10", "-", ">300"],  # to dry
+                [2, "<0.10", "-", "< 400"],  # also to dry
+                [3, ">4.0", ">3.0", "-"],  # to deep
+                [4, ">4.0", "<2.0", "-"],  # also to deep
+                [5, "-", "-", ">400"],  # to salty
+                [6, "-", ">1.5", "-"],  # to fast flowing
+                [7, "-", ">1.5", ">300"],  # alos to fast flowing
+                [8, "0.20:4.0", "0.0:1.5", "0:400"],  # perfect for aquatic plants
+            ],
+            [
+                call(
+                    "For conditions: (MIN_water_depth_mNAP: -, MAX_flow_velocity: -). Gap for variable MAX_chloride in range -inf:400.0\nFor conditions: (MIN_water_depth_mNAP: -, MAX_flow_velocity: >1.5). Overlap for variable MAX_chloride in range 300.0:inf\nFor conditions: (MIN_water_depth_mNAP: 0.20:4.0, MAX_flow_velocity: 0.0:1.5). Gap for variable MAX_chloride in range -inf:0.0\nFor conditions: (MIN_water_depth_mNAP: 0.20:4.0, MAX_flow_velocity: 0.0:1.5). Gap for variable MAX_chloride in range 400.0:inf\nFor conditions: (MIN_water_depth_mNAP: <0.10, MAX_flow_velocity: -). Overlap for variable MAX_chloride in range 300.0:400.0\nFor conditions: (MIN_water_depth_mNAP: -). Overlap for variable MAX_flow_velocity in range 1.5:inf"
+                ),
+                call(
+                    "12 warnings found concerning coverage of the parameters. Only first 6 "
+                    "warnings are shown. See classification_warnings.log file for all warnings."
+                ),
+            ],
         ),
     ],
 )
-def test_feedback_for_criteria_multiple_parameters(
-    criteria_table, expected_warning_msg
-):
+def test_feedback_for_criteria_multiple_parameters(criteria_table, calls):
     """Test if a correct dictionary is parsed into a RuleData object"""
     # Arrange
     contents = dict(
         {
             "name": "testname",
-            "input_variables": ["varA", "varB", "varC", "varD"],
+            "input_variables": [
+                "varA",
+                "varB",
+                "varC",
+                "varD",
+                "MIN_water_depth_mNAP",
+                "MAX_flow_velocity",
```

Comment:
no objection, why change this?

Reply:
was coming from the realistic example marc provided. Was too lazy to change the name, and this gives an impression on how the actual result would look like